Jellyfin.Plugin.OpenSubtitles/OpenSubtitlesHandler/OpenSubtitles.cs

```diff
@@ -127,7 +128,7 @@ public static async Task<HttpResponse> DownloadSubtitleAsync(string url, Cancell
         /// <returns>The list of response data.</returns>
         public static async Task<ApiResponse<IReadOnlyList<ResponseData>>> SearchSubtitlesAsync(Dictionary<string, string> options, string apiKey, CancellationToken cancellationToken)
         {
-            var opts = System.Web.HttpUtility.ParseQueryString(string.Empty);
+            var opts = new NameValueCollection();
```

Comment:
Does this work? I used `HttpUtility.ParseQueryString` because the `ToString` method returns a query string of the params